src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -65,11 +67,15 @@ private static void boot(Connection connection) {
         TemplateOperations templateOperations = new TemplateOperations(context);
         NotificationRestOperations notificationRestOperations = new NotificationRestOperations(context);
         PlatformOperations platformOperations = new PlatformOperations(context);
+        WorkerOperations workerOperations = new WorkerOperations(context);
+
+        // TODO Instantiate PlatformManager
```

Comment:
yep, I think this is the best place for it, since the object can be handed to every part where it is needed.